pkg/catalog/generator.go

```diff
@@ -189,6 +189,9 @@ func (p *Generator) SetOptions(
 	return p
 }
 
+var maxCreatorCount = 240
```

Comment:
Just curious, what made you decide to go with 240 here? I've tested it and it does seem to slow down quite a bit, so wondering how large we can make this.

Also, should this be a constant instead?

Reply:
https://wilsonmar.github.io/maximum-limits/
![image](https://user-images.githubusercontent.com/13619421/85964487-311bee80-b9fd-11ea-9a46-d31f86a8745a.png)

I set it to 240 as the soft limit of my system is 256.